### ISSUE TITLE
Another MOC fork

### ISSUE THREAD

**Issue by tomaszg7** [xdch47/moc#1]
Hi, I found your repo through Gentoo bug report and I wanted to let you know that I've been also developing some things around MOC trunk. I have a bit of fixes in my repo and some new features as well. It is hosted on Gitlab: https://gitlab.com/tomaszg/mocp If you don't have anything against it, I'll cherry-pick some of your commits and include it in my fork (at the very least ffmpeg stuff). You are of course free to do the same with my patches.

**Comment by xdch47** [comment]
yes, of course - I have not see your repo while collecting the other patches around (such as the pulse audio one).
i like that you also find '..' very convenient / usable even so strictly spoken it is kind of misbehave. (https://gitlab.com/tomaszg/mocp/-/commit/0f467b181d1739c7014f8d5e28ae420e56befb8d).

thanks for the hint.

**Comment by tomaszg7** [comment]
Nice, I already included many of your changes in my fork. Pulseaudio needed some small fixes to make it compatible with my other patches.

**Comment by rezso** [comment]
@tomaszg7 unfortunately, you don't use tags, so your repo not packaging-friendly.

**Comment by tomaszg7** [comment]
> @tomaszg7 unfortunately, you don't use tags, so your repo not packaging-friendly.

Well, I never had any need for them and didn't know that anyone else would like to have them. I'm a bit surprised that some packaging schemes depends so much on the presence of tags - I've often seen just git snapshots.

I can easily add some tags, but I didn't think about any versioning scheme. I may just add a tag like 0.0.1 to the current head, or maybe something more tied to upstream versioning, idk...
